scripts/cc-cli/src/commands/registerAddress.ts

```diff
@@ -52,7 +52,7 @@ async function registerAddressAction(options: OptionValues) {
 function handleSuccess(addressRegistered: AddressRegistered) {
   console.log(
     chalk.green(
-      `Address Registered Successfully!: ${addressRegistered.itemId}`,
```

Comment:
Why remove ?

Reply:
I removed it to assert that the final message of stdout was `Address Registered Successfully!`. In hindsight, I should have kept the output the same and checked using a `substring` method. I will revert that change so the output is more useful to our users.